cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,261 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+`Release 3.3.0 <http://docs.cask.co/cdap/3.3.0/index.html>`__
+=============================================================
+
+New Features
+------------
+- `CDAP-4280 <https://issues.cask.co/browse/CDAP-4280>`__ -
+  Added the ability to add metadata to artifacts.
+
+- `CDAP-4289 <https://issues.cask.co/browse/CDAP-4289>`__ -
+  Added RESTful APIs to set and get properties for an artifact.
+
+- `CDAP-4228 <https://issues.cask.co/browse/CDAP-4228>`__ -
+	Added the ability to configure which artifact a Hydrator plugin should use.
```

Comment:
I mean the description for this JIRA is indented, but the description for the JIRA above it is not.
In other words, line 35 has no preceding space, but line 38 does.